### ISSUE TITLE
Licensing + Question

### ISSUE THREAD

**Issue by fakerybakery** [a1600012888/PhysDreamer#2]
Hi,
Thank you for releasing PhysDreamer! Since you mentioned this is based on Gaussian Splatting, I was assuming it would have to be licensed under the same license? Are there any plans to reimplement those parts so it could be an open-source release?
Thanks!

**Comment by a1600012888** [comment]
Yes, most of my code in `physdreamer/gaussian_3d` is a copy-paste from source code of Gaussian Splatting. But, I haven't think about licenses before, thanks for bring this up! 

I currently don't have planes to reimplement the 3D gaussians rendering part. 

Thanks
